lib/routes/pollingRoutes.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express')
+const nmd = require('nano-markdown')
+const _ = require('lodash/fp')
+
+const router = express.Router()
+
+const complianceTriggers = require('../compliance-triggers')
+const configManager = require('../new-config-manager')
+const plugins = require('../plugins')
+const semver = require('semver')
+const state = require('../middlewares/state')
+const version = require('../../package.json').version
+
+function checkHasLightning (settings) {
+  return configManager.getWalletSettings('BTC', settings.config).layer2 !== 'no-layer2'
+}
+
+function createTerms (terms) {
+  if (!terms.active || !terms.text) return null
+
+  return {
+    active: terms.active,
+    title: terms.title,
+    text: nmd(terms.text),
+    accept: terms.acceptButtonText,
+    cancel: terms.cancelButtonText
+  }
+}
```

Comment:
Maybe it better to use the ternary operator here:

    return (!terms.active || !terms.text)
        ? null
        : {
            active: terms.active,
            title: terms.title,
            text: nmd(terms.text),
            accept: terms.acceptButtonText,
            cancel: terms.cancelButtonText
        }`